tg/views.py

```diff
@@ -102,7 +102,7 @@ def draw(self, msg: Optional[str] = None) -> None:
         self.win.clear()
         if not msg:
             return
-        self.win.addstr(0, 0, msg[: self.w])
+        self.win.addstr(0, 0, msg.replace("\n", " ")[: self.w])
```

Comment:
I think we shouldn't do any msg transformation here, just print it as it here
e.g here https://github.com/paul-nameless/tg/pull/110/files#diff-dd82b960530ded14e2504f2515ced08fR565
all message  transformation should be done in functions like `parse_content`, etc

Reply:
The problem with this part of code is that it fails, when we there is a new line in the msg. Because it draws over the status window size.

Reply:
oh, it is StatusView. ok then